sycl/source/detail/graph_impl.cpp

```diff
@@ -1474,6 +1474,7 @@ void exec_graph_impl::updateImpl(std::shared_ptr<node_impl> Node) {
     }
   }
 
+  UpdateDesc.hNewKernel = UrKernel;
```

Comment:
Now that `hNewKernel` is optional, would `nullptr` be a better default until the SYCL-RT feature is implemented.